contracts/BaseLaunchpeg.sol

```diff
@@ -19,16 +19,20 @@ abstract contract BaseLaunchpeg is
     IBaseLaunchpeg,
     ERC721AUpgradeable,
     BatchReveal,
-    OwnableUpgradeable,
+    SafeAccessControlEnumerableUpgradeable,
     ReentrancyGuardUpgradeable,
     ERC2981Upgradeable
 {
     using StringsUpgradeable for uint256;
 
+    /// @notice Role granted to project owners
```

Comment:
just to check - i presume it's fine to change the order of the variables even though this is an upgradeable contract because each collection will be a new deployment, is that right?